packages/venia-ui/lib/components/CartPage/cartPage.css

```diff
@@ -0,0 +1,76 @@
+.root {
+    padding: 1.5rem;
+}
+
+.body {
+    display: grid;
+    gap: 2rem;
+    grid-template-columns: 66% auto;
+    /* The summary grid item spans the entire right column. */
+    grid-template-areas:
+        'items summary'
+        'adjustments summary'
+        'recents summary';
+}
+
+.heading_container {
+    display: flex;
+    align-items: baseline;
+    justify-content: space-between;
+
+    margin-bottom: 1rem;
+}
+
+.heading {
+    line-height: 1.25em;
+}
+
+.items_container {
+    grid-area: items;
+}
+
+.price_adjustments_container {
+    grid-area: adjustments;
+}
+
+.recently_viewed_container {
```

Comment:
Since this is an extension/dynamic location can we just call this `cart_extension_container` and rename the grid area?

Reply:
Do we know for sure that's the case? I wasn't clear.

Reply:
Ah yea, it was discussed when we broke the stories down that its more like a configurable area for things like "recently viewed" or "people like you purchased..." etc.

Reply:
TBH It may be better to simply remove/comment out the html since those components don't exist and I'm not sure we are expected to even implement it within this epic.